### DISCUSSION TITLE
test

### DISCUSSION THREAD

**Discussion by huangkairan** [lencx/gg#3] [Try]
test

**Comment by lencx** [comment]
这个每天定时自动构建一次，并不能实时看到效果，明天就可以看到了。重新发布代码，或者重新运行 actions 也可以触发构建。

**Reply by huangkairan** [reply]
haha 被你发现我在想什么了

**Reply by lencx** [reply]
现在已经可以看到了 https://gg.nofwl.com/issues/3
